src/logger.js

```diff
@@ -1,14 +1,17 @@
+import color_message from './color'
+
 const LOG_LEVELS = {
   debug: 1,
   info: 2,
   silent: 3
 }
 
-export default function (levelName, message) {
+export default function (levelName, message, color) {
```

Comment:
We need 2 instances of class if we do this.

I am only coloring messages pertinent to flaky. These message use `colorLogger`. Other messages that are generated by protractor use `plainLogger`.